interfaces/Portalicious/src/app/utils/locale.ts

```diff
@@ -0,0 +1,85 @@
+import { isDevMode } from '@angular/core';
+
+export const localStorageLocaleKey = 'preferredLanguage';
+
+export enum Locale {
+  en = 'en',
+  nl = 'nl',
+}
+
+export function getLocaleLabel(locale: Locale): string {
+  switch (locale) {
+    case Locale.en:
+      return $localize`English`;
+    case Locale.nl:
+      return $localize`Dutch`;
```

Comment:
```suggestion
      return 'English';
    case Locale.nl:
      return 'Nederlands';
```

I think languages should have their own 'native' name, to aid in 'discoverability'/scanability. (And also, because we currently have them that way.)